### PR TITLE
Add changelog entries for #4869

### PR DESCRIPTION
**Pull request by cognifloyd**
This adds the changelog entries missing from #4869

### CODE REVIEW

**Comment by cognifloyd** [comment]
Hey look here's another recent entry for requests.
https://github.com/StackStorm/st2/blob/21af5544672e9906f856fce66d287a22291098f7/CHANGELOG.rst#L80-L81

That's more what I'm doing so I followed that pattern.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -109,6 +112,10 @@ Fixed
 
   Contributed by Tatsuma Matsuki (@mtatsuma)
 
+* Fix dependency conflicts by updating ``requests`` to latest version (2.23.0). #4869
+* Fix dependency conflicts by updating ``gitpython`` to version 2.23.0 #4869
```
Comment (blag) [inline CHANGELOG.rst]:
```suggestion
* Fix dependency conflicts by updating ``gitpython`` to version 2.1.15 #4869
```

Comment (cognifloyd) [reply CHANGELOG.rst]: doh. copy/paste error. pushing fix...

File: CHANGELOG.rst [hunk 2]
```diff
@@ -109,6 +112,10 @@ Fixed
 
   Contributed by Tatsuma Matsuki (@mtatsuma)
 
+* Fix dependency conflicts by updating ``requests`` to latest version (2.23.0). #4869
+* Fix dependency conflicts by updating ``gitpython`` to version 2.1.15 #4869
+* Fix travis issue that prevented root from writing virtualenv lock files under /home/travis. #4869
```
Comment (armab) [inline CHANGELOG.rst]: @blag We usually don't include Travis, CI/CD, Makefile or any other build-related changes in `st2` Changelog.
What matters is `stackstorm/st2` core platform or whether its behavior changed or not.
And so in this situation, only `pip` changes are relevant for the changelog.

@cognifloyd Ideally if we'll have 1 line in Changelog for that bugfix.

Sorry for confusion.

Comment (cognifloyd) [reply CHANGELOG.rst]: I initially left out the build changes and only mentioned the requirements updates, so what you say makes sense. I changed to meet one Senior Maintainers request, and now I have another Senior Maintainer requesting the opposite. So, @blag, are you ok with @armab's request, with going back to one requirements entry?

Comment (blag) [reply CHANGELOG.rst]: Yes, I'm fine with that.

Comment (cognifloyd) [reply CHANGELOG.rst]: done.

**Comment by armab** [comment]
@cognifloyd @blag Thanks! :+1:

**Pull request merged**